### PR TITLE
Stat cards

### PR DESCRIPTION
**Pull request by nicocapalbo**

### CODE REVIEW

File: pfdp/css/stats.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.stat-container {
+  display: flex;
+  justify-content: space-around;
+}
+
+.stat-card {
+  /* color: grey; */
```
Comment (cecile-dzy-ncl) [inline pfdp/css/stats.css]: delete comments when they are not necessary for code understanding

File: pfdp/css/stats.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  width: 28%;
+}
+
+.stat-number {
+  font-family: "Montserrat", sans-serif !important;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/stats.css]: no need to put the font-family, the body already have it

File: pfdp/css/stats.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: 70px !important;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/stats.css]: `! important` not necessary here, you can delete it

File: pfdp/css/stats.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.stat-text {
+  text-align: center;
+}
+
+
+@media screen and (max-width: 500px) {
+  .stat-container {
+    /* width: 80%; */
```
Comment (cecile-dzy-ncl) [inline pfdp/css/stats.css]: same here for the comment

**Review by cecile-dzy-ncl — CHANGES_REQUESTED** [review]

File: pfdp/css/stats.css (hunk 1 above)
Comment (nicocapalbo) [reply pfdp/css/stats.css]: removed

File: pfdp/css/stats.css (hunk 2 above)
Comment (nicocapalbo) [reply pfdp/css/stats.css]: if i remove it, it reverts to **times** font

Comment (nicocapalbo) [reply pfdp/css/stats.css]: image

File: pfdp/css/stats.css (hunk 3 above)
Comment (nicocapalbo) [reply pfdp/css/stats.css]: removed **!important**

File: pfdp/css/stats.css (hunk 4 above)
Comment (nicocapalbo) [reply pfdp/css/stats.css]: removed

File: pfdp/css/stats.css (hunk 2 above)
Comment (cecile-dzy-ncl) [reply pfdp/css/stats.css]: ok let's keep it for now then

**Review by cecile-dzy-ncl — APPROVED** [review]

**Pull request merged**